src/Dependencies/PooledObjects/ArrayBuilder.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.PooledObjects
 {
     [DebuggerDisplay("Count = {Count,nq}")]
     [DebuggerTypeProxy(typeof(ArrayBuilder<>.DebuggerProxy))]
-    internal sealed partial class ArrayBuilder<T> : IReadOnlyCollection<T>, IReadOnlyList<T>
+    internal sealed partial class ArrayBuilder<T> : IReadOnlyCollection<T>, IReadOnlyList<T>, IDisposable
```

Comment:
>IDisposable [](start = 94, length = 11)

Are we using this interface anywhere? I'm not against the change but allowing `using` statements for builders has been suggested (unsuccessfully) before.

Reply:
Whoops, I was experimenting with this, I thought in a different repo. Must have missed it.

---
In reply to: [274100294](https://github.com/dotnet/roslyn/pull/34429#discussion_r274100294) [](ancestors = 274100294)